service/src/main/kotlin/fi/espoo/evaka/daycare/controllers/TermsController.kt

```diff
@@ -47,6 +52,76 @@ class TermsController(private val accessControl: AccessControl) {
         return db.connect { dbc -> dbc.read { it.getPreschoolTerms() } }
     }
 
+    @PostMapping("/club-terms")
+    fun createClubTerm(
+        db: Database,
+        user: AuthenticatedUser,
```

Comment:
consider using `user: AuthenticatedUser.Employee` --> then these clients won't be generated for employee-mobile-frontend